rpc/client.go

```diff
@@ -157,6 +157,19 @@ type ClientInformational interface {
 
 	// GetChainspec returns the raw bytes of the chainspec.toml, accounts.toml and global_state.toml files as read at node startup.
 	GetChainspec(ctx context.Context) (InfoGetChainspecResult, error)
+
+	// GetValidatorRewardByEraID returns the reward for a given era and a validator
+	GetValidatorRewardByEraID(ctx context.Context, validator keypair.PublicKey, eraID uint32) (InfoGetRewardResponse, error)
+	// GetValidatorRewardByBlockHash returns the reward for a given block hash and a validator
+	GetValidatorRewardByBlockHash(ctx context.Context, validator keypair.PublicKey, blockHash string) (InfoGetRewardResponse, error)
```

Comment:
Usually the SDK has `*Result` suffix instead of `Response`. i.e.: `InfoGetRewardResult`.